test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js

```diff
@@ -23,23 +23,22 @@ import selectionModule from 'diagram-js/lib/features/selection';
 
 import coreModule from 'bpmn-js/lib/core';
 import modelingModule from 'bpmn-js/lib/features/modeling';
-import propertiesPanelModule from 'bpmn-js-properties-panel';
+import { BpmnPropertiesPanelModule, ZeebePropertiesProviderModule } from 'bpmn-js-properties-panel';
 
 import zeebeModdleExtensions from 'zeebe-bpmn-moddle/resources/zeebe';
 
-import propertiesProviderModule from 'lib/camunda-cloud/features/properties-provider';
 import zeebeModelingModules from 'lib/camunda-cloud/features/modeling';
 
 import diagramXML from './process-call-activities.bpmn';
 
 
-describe('camunda-cloud/features/modeling - UpdatePropagateAllChildVariablesBehavior (integration tests)', function() {
+describe.only('camunda-cloud/features/modeling - UpdatePropagateAllChildVariablesBehavior (integration tests)', function() {
```

Comment:
This does not belong here

Reply:
I am wondering how it was ignored by the linter 🤯

Reply:
all right, I added this and then removed :/